client/src/models/enums/selfInvestigationStatuses.ts

```diff
@@ -0,0 +1,20 @@
+enum SelfInvestigaionStatuses {
+    UNOPENED = 0,
+    IN_PROGRESS = 1,
+    COMPLETED = 2
+}
+
+export const getInvestigationStatusName = (status : number) => {
+    switch(status) {
+        case SelfInvestigaionStatuses.UNOPENED :
+            return 'לא נפתח'
+        case SelfInvestigaionStatuses.IN_PROGRESS :
+            return 'בתהליך'
+        case SelfInvestigaionStatuses.COMPLETED :
+            return 'הושלם'
+        default : 
+            return 'לא ידוע'
+    }
+}
```

Comment:
add object that its keys are the enum and the value is an object that contains the name

Reply:
added [here ](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1093/commits/1dbd9ba715185239e98a27e85d126a7c2314668a)